src/CabanaPD_Solver.hpp

```diff
@@ -669,6 +669,131 @@ class SolverFracture
     using base_type::print;
 };
 
+template <class MemorySpace, class InputType, class ParticleType,
+          class ForceModel, class ContactModel>
+class SolverContact
+    : public SolverFracture<MemorySpace, InputType, ParticleType, ForceModel>
+{
+  public:
+    using base_type =
+        SolverFracture<MemorySpace, InputType, ParticleType, ForceModel>;
+    using exec_space = typename base_type::exec_space;
+    using memory_space = typename base_type::memory_space;
+
+    using particle_type = typename base_type::particle_type;
+    using integrator_type = typename base_type::integrator_type;
+    using comm_type = typename base_type::comm_type;
+    using neighbor_type = typename base_type::neighbor_type;
+    using force_model_type = ForceModel;
+    using force_type = typename base_type::force_type;
+    using neigh_iter_tag = Cabana::SerialOpTag;
```

Comment:
I assume you'll mean to expose `neigh_iter_tag` to allow for `TeamOpTag` at a later time.

Reply:
Yes, but outside the scope of this PR